cypress/e2e/App.cy.tsx

```diff
@@ -7,8 +7,21 @@ describe('Root.cy.tsc', () => {
     cy.visit('/');
   });
 
-  it('hello world', () => {
-    cy.getBySelector('hello-world-p').contains('Hello World');
+  it('render Header', () => {
+    cy.getBySelector('header').should('exist');
+  });
+
+  it('display images in Header', () => {
+    cy.getBySelector('header-grid')
+      .find('img')
+      .should('be.visible')
+      .each((img: { naturalWidth: number }[]) => {
```

Comment:
it would be better to create separate tests for both images, please add data-cy for each one of them